server/services/hubMenu.js

```diff
@@ -1,27 +1,6 @@
-const createHubMenuService = repository => {
-  function primaryMenu() {
-    return repository.primaryMenu();
-  }
-
-  function tagsMenu() {
-    return repository.tagsMenu();
-  }
-
-  function allTopics(prisonId) {
-    return repository.allTopics(prisonId);
-  }
-
-  function categoryMenu({ category, prisonId }) {
-    return repository.categoryMenu({ category, prisonId });
-  }
-
-  return {
-    tagsMenu,
-    primaryMenu,
-    categoryMenu,
-    allTopics,
-  };
-};
+const createHubMenuService = repository => ({
+  allTopics: prisonId => repository.allTopics(prisonId),
```

Comment:
these other methods weren't being used - I'll convert this into a `TopicService` class in a follow up